GPy/core/sparse_gp_mpi.py

```diff
@@ -88,9 +88,9 @@ def optimizer_array(self, p):
     def optimize(self, optimizer=None, start=None, **kwargs):
         self._IN_OPTIMIZATION_ = True
         if self.mpi_comm==None:
-            super(SparseGP_MPI, self).optimize(optimizer,start,**kwargs)
+            ret = super(SparseGP_MPI, self).optimize(optimizer,start,**kwargs)
```

Comment:
I'm not sure if this is the right course here. I've never used an MPI implementation so I'm unsure about how this would work. These modifications enable this class to be used with the python multiprocessing pool in the same way as the base GP class.